SimG4CMS/Calo/interface/HFNoseSD.h

```diff
@@ -44,7 +44,7 @@ class HFNoseSD : public CaloSD, public Observer<const BeginOfJob *> {
   const HGCalDDDConstants*               hgcons_;
   std::unique_ptr<HFNoseNumberingScheme> numberingScheme_;
   std::unique_ptr<HGCMouseBite>          mouseBite_;
-  const std::string                      nameX_ = "HFNoseSensitive";
+  const std::string                      nameX_ = "HGCalHFNoseSensitive";
```

Comment:
@bsunanda , I would say it is unnecessary overhead to add nameX_, there is no need to have two names - one from GetName() another from local variable. I would suggest to drop this class member.

Reply:
It is the name of the Geometry object. We cannot get that from GetName().